Sources/Hummingbird/Server/RequestContext.swift

```diff
@@ -68,8 +59,8 @@ public struct HBCoreRequestContext: Sendable {
     public init(
         applicationContext: HBApplicationContext,
         eventLoop: EventLoop,
-        logger: Logger,
-        allocator: ByteBufferAllocator = .init()
+        allocator: ByteBufferAllocator,
```

Comment:
Any particular reason for the order change?

Reply:
Konrad has always made it a standard that the logger is the last parameter in a function call